docs/kyma/04-03-cluster-installation.md

```diff
@@ -117,13 +117,15 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
     az group create --name $RS_GROUP --location $REGION
     ```
 
-3. Create a [service principle](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal#manually-create-a-service-principal) on Azure. Create a TOML file with the Azure Client ID, Client Secret, Subscription ID, and Tenant ID:
-
-    ```toml
-    CLIENT_ID = {YOUR_APP_ID}
-    CLIENT_SECRET = {YOUR_APP_PASSWORD}
-    SUBSCRIPTION_ID = {YOUR_SUBSCRIPTION_ID}
-    TENANT_ID = {YOUR_TENANT_ID}
+3. Create a [service principle](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal#manually-create-a-service-principal) on Azure. Create a JSON file with the Azure Client ID, Client Secret, Subscription ID, and Tenant ID:
```

Comment:
```suggestion
3. Create a [service principal](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal#manually-create-a-service-principal) on Azure. Create a JSON file with the Azure Client ID, Client Secret, Subscription ID, and Tenant ID:
```

Reply:
I know you have not changed that but I think it should be `principal` in the first place :)